app/mediaAnalyser/AvSoundFile.cpp

```diff
@@ -71,7 +71,7 @@ AvSoundFile::AvSoundFile(const std::vector<AudioElement>& arrayToAnalyse)
         // Get data from audio stream
         const avtranscoder::AudioProperties* audioProperties = reader->getSourceAudioProperties();
         const int nbChannels = channelIndex == -1 ? audioProperties->getNbChannels() : 1;
-        _inputNbChannels.push_back(nbChannels);
+        _inputNbChannels.push_back(std::min(nbChannels, 5)); // skip LRE
```

Comment:
We ignore the next ones. This was already the case, but it crashes with this application. This commit fixes the crash. But maybe the right solution is to let the user choose the list of channels he wants to analyse...